src/components/TextInput/TextInput.tsx

```diff
@@ -49,6 +53,8 @@ const TextInput: React.FC<TextInputProps> = ({
         onChange={handleChange}
         name={name}
         disabled={disabled}
+        onMouseOver={onMouseOver}
+        onMouseOut={onMouseOut}
```

Comment:
**question:** Why this way? Hover was not working?

Reply:
Imho I didn't come up to this idea. I've already fixed it to your version ;)